src/frontend/src/lib/components/dapps/DappPromoBanner.svelte

```diff
@@ -1,27 +1,23 @@
 <script lang="ts">
 	import Button from '$lib/components/ui/Button.svelte';
 	import Img from '$lib/components/ui/Img.svelte';
+	import ImgBanner from '$lib/components/ui/ImgBanner.svelte';
 	import { i18n } from '$lib/stores/i18n.store';
 	import type { FeaturedOisyDappDescription } from '$lib/types/dapp-description';
 	import { replacePlaceholders } from '$lib/utils/i18n.utils';
 
 	export let dAppDescription: FeaturedOisyDappDescription;
 </script>
 
-<article class="relative flex h-64 items-end overflow-hidden rounded-2xl">
+<article class="relative flex items-end overflow-hidden rounded-2xl">
 	{#if dAppDescription.screenshots.length > 0}
-		<div class="absolute inset-0 bg-brand-subtle-alt">
-			<Img
-				fitHeight={true}
-				height="100%"
-				width="100%"
-				styleClass="object-cover"
-				src={dAppDescription.screenshots[0]}
-				alt={replacePlaceholders($i18n.dapps.alt.website, { $dAppName: dAppDescription.name })}
-			/>
-		</div>
+		<ImgBanner
+			styleClass="bg-brand-subtle-alt h-64 max-h-64"
```

Comment:
I think better to keep `bg-brand-subtle-alt` class in the parent `<div>`

Reply:
I mean like this we do not need a parent `<div>` anymore, which i think is cleaner.

Reply:
but then you need to provide `styleClass`, while right now you could provide only `maxSize` prop for example (`large`, `small`, `none`)

Reply:
I don't think it makes sense to replace the `styleClass` with a prop `maxSize` that takes values like: `large`, `small`, `none`. At some points we need the `<ImgBanner` with a max height of 64px and at other points with a max height of 56px. This means it would be necessary to provide the max height in px (for example: `maxSize: 64px`) instead of `large`, etc. This would be more or less the same as with the styleClass.

Reply:
I think i didn't understand: you are using `styleClass` either for `max-h-64` and `max-h-56` or `none` (where the numbers are not px, but a size value). Why not just using it as defined multiple choice prop?

Reply:
@AntonioVentilii Now i moved the `bg-brand-subtle-alt` to the parent `<div>` and this requires that `max-h-64` is also moved to the parent `<div>`. So we do not need the `styleClass` anymore on the '<ImgBanner'.

Reply:
ah perfect! sorry for the back and forth, but it seems cleaner in a way